### PR TITLE
misc(release): plug release registry

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
### Description

Plugs the release registry into release.yaml

> NOTE:

### Checklist

- [ ] [CHANGELOG.md](https://github.com/sourcegraph/sourcegraph/blob/main/CHANGELOG.md) updated
- [ ] [K8s Upgrade notes updated](https://github.com/sourcegraph/sourcegraph/blob/main/doc/admin/updates/kubernetes.md)
- [ ] Sister [deploy-sourcegraph-k8s](https://github.com/sourcegraph/deploy-sourcegraph-docker) change:
- [ ] Sister [deploy-sourcegraph-docker](https://github.com/sourcegraph/deploy-sourcegraph-docker) change:
- [ ] All images have a valid tag and SHA256 sum
- [x] I acknowledge that [deploy-sourcegraph-k8s](https://github.com/sourcegraph/deploy-sourcegraph-k8s) is now the preferred Kubernetes deployment repository

### Test plan

Presumed working, copied from deploy-sourcegarph

### CODE REVIEW

**Comment by Chickensoupwithrice** [comment]
Waiting for testing before merge: https://github.com/sourcegraph/sourcegraph/issues/61078#issuecomment-2082883491

**Review by DaedalusG — APPROVED** [review]

**Comment by BolajiOlajide** [comment]
Hey @Chickensoupwithrice 

I totally forgot about this PR and created a new one. I'm going to close this one out now.
My bad!

**Pull request closed without merging**
